wifi_enable.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+echo "This command must be run as SuperUser and using Bash"
+echo "Please give this Pi a unique ID (for the WiFi Access Point):"
+Pass_Phrase=pi2733hotspot
+read Pi_ID
+while true; do
+  echo "Enable Access Point [Y/n]"
+  read enable_ap
+  if [[ $enable_ap = "" || $enable_ap = "y" ]]; then
```

Comment:
I not entirely sure about this (I really don't know much bash), but would it be a good idea to check for both upper and lower case 'y' rather than just lowercase?

Reply:
Yes it would be, but it didn't seem like too big of an issue. I'll add it though.